src/components/pill/pill.spec.js

```diff
@@ -362,16 +361,133 @@ describe('Pill', () => {
       }, TestRenderer.create).toJSON();
       assertStyleMatch({
         borderRadius: '12px',
-        fontSize: '14px',
+        fontSize: '10px',
         fontWeight: '600',
         position: 'relative',
         top: '-1px',
-        padding: '2px 8px 2px 8px',
+        padding: '4px 8px 2px 8px',
         margin: '0px 8px 16px 0px'
       }, wrapper);
     });
   });
 
+  describe('classic theme', () => {
+    const renderBase = (props, renderer = mount) => {
+      return renderer(
+        <ThemeProvider theme={ classicTheme }>
+          <Pill { ...props } />
+        </ThemeProvider>
+      );
+    };
+
+    it('switches to use the modern classic theme', () => {
+      const wrapper = renderBase({
+        children: 'My Text',
+        theme: classicTheme
+      }, TestRenderer.create).toJSON();
+      assertStyleMatch({
+        borderRadius: '10px',
+        fontSize: '12px',
+        minWidth: '24px'
+      }, wrapper);
+    });
+  });
+
+  describe('base theme', () => {
+    const renderBase = (props, renderer = mount) => {
```

Comment:
Same here. Also, there is already a describe block for base theme further up, so these tests should be moved under that.

Reply:
resolved